### PR TITLE
Adding DoD to the issue template

### PR DESCRIPTION
**Pull request by kasiakepka**
Adding DoD to the issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ assignees: ''
 
 <!-- Explain why we should add this feature. Provide use cases to illustrate its benefits. -->
 
+**DoD:**
+- [ ] provide documentation
+- [ ] provide unit tests
+- [ ] test on SKR
+- [ ] verify [Get started guides](https://kyma-project.io/docs/kyma/latest/02-get-started/)
```
Comment (dariusztutaj) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]: Is it needed for api-gateway?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -20,6 +20,14 @@ assignees: ''
...
 <!-- Explain why we should add this feature. Provide use cases to illustrate its benefits. -->
 
+**DoD:**
+- [ ] provide documentation
+- [ ] provide unit tests
+- [ ] test on SKR
+- [ ] verify [Get started guides](https://kyma-project.io/docs/kyma/latest/02-get-started/) 
+- [ ] verify resource limits
+- [ ] followup issue
+
```
Comment (strekm) [inline .github/ISSUE_TEMPLATE/feature_request.md:30]: maybe we can add `bump in kyma`? @kyma-project/goat what do you think?

**Review by dariusztutaj — APPROVED** [review]
/meow

**Review by strekm — APPROVED** [review]
/meow

**Pull request merged**